test/common/network/matching/inputs_integration_test.cc

```diff
@@ -171,6 +208,25 @@ TEST_F(InputsIntegrationTest, ApplicationProtocolInput) {
   EXPECT_TRUE(result.on_match_.has_value());
 }
 
+TEST_F(InputsIntegrationTest, FilterStateInput) {
+  std::string key = "filter_state_key";
+  std::string value = "filter_state_value";
+  initializeFilterStateCase("FilterStateInput", key, value);
+
+  StreamInfo::FilterStateImpl filter_state(StreamInfo::FilterState::LifeSpan::FilterChain);
+  filter_state.setData(key, std::make_shared<Router::StringAccessorImpl>(value),
+                       StreamInfo::FilterState::StateType::Mutable,
+                       StreamInfo::FilterState::LifeSpan::Connection,
+                       StreamInfo::FilterState::StreamSharing::SharedWithUpstreamConnection);
```

Comment:
Omit this last parameter SharedWithUpstreamConnection since it's irrelevant.

Reply:
done